### ISSUE TITLE
Question: traefik setup to make multiple containers accessible in local network.

### ISSUE THREAD

**Issue by bsr203** [microsoft/navcontainerhelper#2614]
Hello,

TLDR: Request for instructions to setup a simple traefik proxy to access all nav containers on localhost.

I was previously exposing container ports during `New-BcContainer` to make multiple containers in local network.

```
	-WebClientPort 8082 `
	-FileSharePort 8280 `
	-ManagementServicesPort 7245 `
	-ClientServicesPort 7246 `
	-SoapServicesPort 7247 `
	-ODataServicesPort 7248 `
	-DeveloperServicesPort 7249 `
    	-updateHosts `
    	-publishPorts @(8082,8280,7245,7246,7247,7248,7249)
```

.@freddydk suggested the use of traefik, and  as I started using more containers, it became hard to remember all the ports and managing them without conflict.

I am not an expert on docker and windows, and also never used `traefik`. My requirements are simple

- create multiple containers  using `New-BcContainer` with `containerName` say `bc1`, `bc2`, `bc3`,.. 
- make them available in local network and able to develop against (so all ports needs to be exposed and able to debug,..)

When try to follow the [official installation instruction](https://doc.traefik.io/traefik/getting-started/quick-start/#launch-traefik-with-the-docker-provider), I got this error.

```
PS C:\Users\bsr\work\docker\traefik> docker-compose up -d reverse-proxy
Creating network "traefik_default" with the default driver
Pulling reverse-proxy (traefik:v2.8)...
v2.8: Pulling from library/traefik
ERROR: no matching manifest for windows/amd64 10.0.22000 in the manifest list entries
```
While trying to troubleshoot, I saw few items in this repository, which make me doubt my approach.
- Is traefik 2.0 is supported? I see an [initial pull req](https://github.com/microsoft/navcontainerhelper/pull/2169). so not sure should use it.
- is official traffic installation supported. I see a script `Setup-TraefikContainerForBcContainers` and `useTraefik` switch. So may be I should be using these
- Open issues https://github.com/microsoft/navcontainerhelper/issues/1895

It will be great if we can document (or point to a blog post) how to setup traefik, so we can add a container quickly and access it in localhost for development.

Thanks
bsr.

**Comment by freddydk** [comment]
You can use the command in BcContainerHelper called:

Setup-TraefikContainerForBcContainers
